docs/tasks/debug-application-cluster/run-command-container-kubectl-exec.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caesarxuchao
+- mikedanese
+title: Running Commands in a Container using kubectl exec
+---
+
+{% capture overview %}
+
+This page shows how to use `kubectl exec` to run commands in a Container.
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## Using kubectl exec to view the environment variables of a Container
+
+Kubernetes exposes
```

Comment:
a bit awkward:

Containers can access cluster information, such as Service endpoints and ConfigMaps, through environment variables.  These environment variables can be printed by executing "env" on a running container.